CHANGELOG.md

```diff
@@ -45,11 +58,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Support new versions of ethereumjs/tx ([#88](https://github.com/metamask/eth-trezor-keyring/pull/88))
 
-[Unreleased]: https://github.com/metamask/eth-trezor-keyring/compare/v1.1.0...HEAD
-[1.1.0]: https://github.com/metamask/eth-trezor-keyring/compare/v1.0.0...v1.1.0
-[1.0.0]: https://github.com/metamask/eth-trezor-keyring/compare/v0.10.0...v1.0.0
-[0.10.0]: https://github.com/metamask/eth-trezor-keyring/compare/v0.9.1...v0.10.0
-[0.9.1]: https://github.com/metamask/eth-trezor-keyring/compare/v0.9.0...v0.9.1
-[0.9.0]: https://github.com/metamask/eth-trezor-keyring/compare/v0.8.0...v0.9.0
-[0.8.0]: https://github.com/metamask/eth-trezor-keyring/compare/v0.7.0...v0.8.0
-[0.7.0]: https://github.com/metamask/eth-trezor-keyring/releases/tag/v0.7.0
+[Unreleased]: https://github.com/MetaMask/eth-trezor-keyring/compare/v2.0.0...HEAD
+[2.0.0]: https://github.com/MetaMask/eth-trezor-keyring/compare/v1.1.0...v2.0.0
```

Comment:
Well... that's curious. I wonder why it's capitalized here now.

Reply:
The create-release-branch action wants it to be upper-case, but the linter wants it to be lower-cased. The linter us using the field in `package.json` (which is lower-cased), but the create-release-branch action must be using something else.

It's a bit silly for the linter to care, and they should both be using the same source, but the short term solution might be to update the package.json field to use proper capitalization.

In the meantime we can undo these changes here to keep the linter happy and get this release out

Reply:
Just pushed the case correction.